pkgs/top-level/all-packages.nix

```diff
@@ -16383,6 +16383,8 @@ with pkgs;
 
   poetry = callPackage ../tools/package-management/poetry { };
 
+  poetry-plugin-up = callPackage ../tools/package-management/poetry-plugin-up { };
```

Comment:
This should be in `python-packages.nix`.

Reply:
Why? It's not going to be useful with any Python version but the one Poetry is built with.

Reply:
It could also live at `poetry.plugins` or something like that.

Reply:
I thought about that but I'm not sure if it's worth adding another scope when we only have one package to put under it.

Reply:
It is. We can ensure that plugins use the same Python package set in two ways: the `plugins` argument becomes a function that is passed `python.pkgs` (see home-assistant) and plugins are packaged in `python3Packages`, or plugins live in `poetry.plugins` where they're automatically built with the correct Python package set.